src/main/build/version.h

```diff
@@ -36,3 +36,5 @@ extern const char* const buildDate;  // "MMM DD YYYY" MMM = Jan/Feb/...
 
 #define BUILD_TIME_LENGTH 8
 extern const char* const buildTime;  // "HH:MM:SS"
+
+#define MSP_API_VERSION_STRING STR(API_VERSION_MAJOR) "." STR(API_VERSION_MINOR)
```

Comment:
Maybe add something like `MSP API: ` to the string to make it explicit what the number is.

Reply:
Could you elaborate? `API_VERSION_MAJOR` and `API_VERSION_MINOR` are already defined in `msp/msp_protocol.h`.  This is just referencing those variables. Should I rename those variables everywhere?